### ISSUE TITLE
Couldn't build project in local IDE

### ISSUE THREAD

**Issue by th1nkgr33n** [e-Spirit/FSDevTools#45]
Hey folks,

I wanna build a new or improve existing commands but I'm not able to build the project on my local IDE (Intellij) with gradle.

The problem here are 2 things combined in 1 main reason:
- Missing credentials for espirit artifactory -

Problems in Detail
1. Missing dependency for fs-server.jar (I can fix this by maintain these files in my own nexus)
2. Missing dependency for de.espirit.process

This second dependency is not hosted here on GitHub and not somewhere else. Where could I get it?
Is it possible to gain a community access to your artifactory? This request exists sind several years by the community and if you want to improve the so called DX you have to do this.

But for now, where could I get the **de.espirit.process**-dependency?

Kind Regards
Olli

**Comment by th1nkgr33n** [comment]
And another thing is, that your develop documentation is related to maven. You switch to gradle a couple of month ago. Will the documentation will also be updated in near future?

**Comment by sosswa** [comment]
Thanks Olli for your engagement.

Access to the e-Spirit artifactory was announced ad Devcon 2019, over a year ago. Nothing happened till now. I'm with you that this would be a great step in the DX.

**Comment by cr4igo** [comment]
Jo. Did someone mentioned the artifactory?! Just give it a little pump-up.
